src/lib/components/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { Badge } from "./Badge";
+
+const meta = {
+  title: "Base/Badge",
+  component: Badge,
+  parameters: {
+    layout: "centered",
+    controls: { exclude: ["alt"], sort: "requiredFirst" },
+  },
+  tags: ["autodocs"],
+  args: {
+    label: "label",
+  },
+  argTypes: {
+    icon: {
+      control: "text",
+    },
+    reverse: {
+      defaultValue: false,
+      control: "boolean",
+    },
+    size: {
+      options: ["small", "large"],
+      control: { type: "radio", default: "large" },
+    },
+  },
+} satisfies Meta<typeof Badge>;
+
+type Story = StoryObj<typeof meta>;
+export const Default: Story = {};
+
+export const WithIcon: Story = {
+  args: {
+    icon: "http://cdn.onlinewebfonts.com/svg/img_182127.png",
```

Comment:
Could you please use this icon in the white version somehow: https://fonts.google.com/icons?selected=Material%20Icons%3Aerror_outline%3A